lib/rubocop/cop/style/format_string.rb

```diff
@@ -48,6 +48,10 @@ class FormatString < Cop
         }
         PATTERN
 
+        def_node_matcher :variable_argument?, <<-PATTERN
+          (send {str dstr} :% send_type?)
```

Comment:
`send_type?` is not enough to avoid the problem. Because it does not cover a variable.
For example

```ruby
a = something()
puts "%f" % a
```


So I think we should use `literal?` method instead of `send_type?`.

Reply:
Thanks for this case. But node here is `lvar` and `literal?` does not work. I have added this test case and modified code accordingly.